modules/core/migrations/v7/genesis_test.go

```diff
@@ -0,0 +1,158 @@
+package v7_test
```

Comment:
should we add a test to make sure migration passes with the interfaces already registered (only if it is a possible concern ofc!)

Reply:
this is already the case. I added a comment in the testing code

Reply:
ah thank you I missed that! 👍